src/Pos/ReportServer/FileOps.hs

```diff
@@ -48,14 +50,15 @@ data LogsHolder = LogsHolder
 -- where data will be stored.
 genReportPath :: UTCTime -> ReportInfo -> FilePath
 genReportPath curTime ReportInfo{..} =
-    date </> repType </> time
+    case rReportType of
+        RCustomReport {} -> "custom-reports" </> date
+        _                -> date </> repType </> time
   where
     repType = case rReportType of
                   RCrash _            -> "crash"
                   RError _            -> "error"
                   RMisbehavior{}      -> "misbehavior"
                   RInfo _             -> "info"
-                  RCustomReport _ _ _ -> "customreport"
```

Comment:
So now this `case` is non-exhaustive.

Reply:
Yes, it's easy to refactor the function so this case is not present. But putting `error "repType is not ever called with RCustomReport"` is fine too, i guess. It just shouldn't be ignored.